sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Release History
 
-## 4.1.2 (Unreleased)
+## 4.1.2-beta (Unreleased)
```

Comment:
update version in changelog 4.2.0-beta.1

Reply:
updated